src/lib/data.ts

```diff
@@ -645,6 +649,10 @@ function load_from_local_storage_and_sync() {
                 const key = full_key.substr(store_prefix.length);
                 try {
                     remote_store[key] = JSON.parse(localStorage.getItem(full_key)) as RemoteKV;
+                    if (remote_store[key].replication !== Replication.LOCAL_OVERWRITES_REMOTE) {
```

Comment:
I think this needs to be `remote_store[key].replication === Replication.REMOTE_OVERWRITES_LOCAL` specifically doesn't it? We wouldn't want this to trigger with `REMOTE_ONLY`

Reply:
Hmm - you are right again!

I did think about that, and I thought "oh, remote only is about when we are setting the value", just writing it out there, but of course it would come bouncing back through here.

What is the rationale for REMOTE_ONLY?

Reply:
... if we don't load the remote value here, when do we ever load it for this account?

Reply:
We do load it, it's set in `remote_store`, which is what is used if `store` isn't set. When `get` is called, it first checks `store`, if nothing is in there it checks `remote_store`, if nothing is in there it uses the default value.

The intent with `REMOTE_ONLY` was to be able to effectively set defaults, so like default theme is black, but don't force other devices to change if they were set to white locally. Come to thing of it, I'm not sure this is any different than `LOCAL_OVERWRITES_REMOTE`.. I'd have to ponder on that a bit more, I've either forgotten the distinction, or erroneously thought there was a distinction before. But regardless, seems like we should be exact with this case.

Reply:
Oh, cunning, right!

It's not the same as LOCAL_OVERRIDES_REMOTE, because it means you can say "if I ever add more devices, I want them to have this value that's different to any value currently set on my devices", wheras with LOCAL_OVERRIDES_REMOTE you'd get that value on your local device as a result of this action.

It's hard to imagine a direct case for this (a user thinking along those lines) but maybe it has an application where we want to do something tricky on their behalf...